### ISSUE TITLE
学长，你有没有试过把Multi-Label转换为多个二分类？

### ISSUE THREAD

**Issue by goodluck110706112** [RandolphVI/Multi-Label-Text-Classification#20]
比如，把标签：[1,0,0]转换成三个结果：1,0,0，每个结果都是每个class中的二分类的预测结果，不知道这种效果和你的通过阈值来判断Multi-Label的方法，哪个好？
谢谢~

**Comment by RandolphVI** [comment]
@goodluck110706112 

你好，不是很能理解的你的前提表述。阈值划分只是选取规则，我猜你的意思是把多标签问题转换成多个二分类问题？如果是这样的话，你多个二分类问题在最后结果选取上还是会面临阈值选择的问题，比如 > 0.5 会记为 1，反之则记为 0。

**Comment by RandolphVI** [comment]
@goodluck110706112 

另外，现在 Multi-label 最普遍最通常的做法就是将多标签分类问题转换成多个二分类问题来算平均
标签 loss，你可以详细地去了解一下我代码中使用的 Loss 函数`sigmoid_cross_entropy_with_logits()`，应该会对你有所启发。

**Comment by goodluck110706112** [comment]
好的，我去试试，谢谢学长~
